roles/pulp_common/vars/CentOS-7.yml

```diff
@@ -9,3 +9,5 @@ pulp_preq_packages:
   - gcc                # For psycopg2
   - make               # For make docs
 pulp_python_interpreter: /usr/bin/python3.6
+pulp_common_python_cryptography:
```

Comment:
I cannot find any collection-specific guidelines on this (or on our use case of having a common role), but I think we should name this `pulp_python_cryptography`.

The reason being that "common" is implied in the generic "pulp" prefix, and it's less verbose.

It might also be moved to another role later, if only that role needs it.